js/components/closed-caption-multi-audio-menu/closedCaptionMultiAudioMenu.js

```diff
@@ -0,0 +1,128 @@
+var React = require('react');
+var ReactDOM = require('react-dom');
+var Utils = require('../utils');
+var classnames = require('classnames');
+var CONSTANTS = require('../../constants/constants');
+var Tab = require('./tab');
+var MultiAudioTab = require('./multiAudioTab');
+
+var ClosedCaptionMultiAudioMenu = React.createClass({
+  componentDidMount: function() {
+    var multiAudioCol = ReactDOM.findDOMNode(this.refs.multiAudioCol);
+    var closeCaptionsCol = ReactDOM.findDOMNode(this.refs.closeCaptionsCol);
+    if (multiAudioCol && closeCaptionsCol && typeof Utils.getMaxElementWidth === 'function') {
+      var maxWidth = Utils.getMaxElementWidth(multiAudioCol, closeCaptionsCol);
+      multiAudioCol.style.width = maxWidth + 'px';
+      closeCaptionsCol.style.width = maxWidth + 'px';
+    }
+  },
+  /**
+   *
+   * @param languages {Array} - list of available languages
```

Comment:
Can you rename this param to languageList? It will be very easy to make a typo and use the wrong variable and then we'll spend 30 minutes debugging it.

It's also part of the coding standards.

Reply:
Thank you for the review! Yeah, I think it's better to adjust the naming to follow the standarts, will do.

Reply:
\+